server/components/form-settings/form-settings.js

```diff
@@ -115,6 +114,8 @@ class FormSettings {
   formHandler() {
     this.closeNotification()
 
+    const disabledArray = [1, 6, 7]
```

Comment:
What is the function of this "disabledArray"? just curious

Reply:
I was wondering about that too